packages/components/src/index.js

```diff
@@ -63,14 +69,28 @@ export { default as ToolbarButton } from './toolbar-button';
 export { default as Tooltip } from './tooltip';
 export { default as TreeSelect } from './tree-select';
 export { default as IsolatedEventContainer } from './isolated-event-container';
-export { createSlotFill, Slot, Fill, Provider as SlotFillProvider } from './slot-fill';
+export {
```

Comment:
Hmm! This happened from my editor's auto format, which leverages ESLint + Prettier from the project's config.
I can revert this if necessary!